### PR TITLE
Feature/wf 544 c linklist

### PR DESCRIPTION
**Pull request by adrianburtscher**
## Pull request
Adds c-linklist component
 
### Ticket
[https://winterhalter-fenner.atlassian.net/browse/WF-544](https://winterhalter-fenner.atlassian.net/browse/WF-544)
 
### Browser testing
- [Product Detail Page](http://localhost:8080/styleguide/product-detail)
 
### Checklist
- [x] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [ ] Did run automated tests and linters
- [ ] Did review code and documentation
- [ ] Tested all links in project relevant browsers
- [ ] Tested all links in different screen sizes
- [ ] Re-assign ticket to developer

### CODE REVIEW

File: app/components/c-linklist.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+</template>
+
+<script>
+  export default {
+    name: 'c-linklist',
+    // components: {},
+    // mixins: [],
+
+    /**
+     * Defines the items, displayed in the list.
```
Comment (geroldboehler) [inline app/components/c-linklist.vue]: - [ ] Move documentation to the property

File: app/components/c-linklist.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <ul :class="b('list')">
+    <li v-for="item in items" :class="b('item')" :key="item.key">
+      <a :class="b('link')" :href="item.value" :title="item.key">
```
Comment (geroldboehler) [inline app/components/c-linklist.vue]: - [ ] Add target="_blank"

File: app/components/c-linklist.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <ul :class="b('list')">
```
Comment (geroldboehler) [inline app/components/c-linklist.vue]: - [ ] No need for a block, just use b('')

File: app/components/c-product-detail.vue [hunk 4]
```diff
@@ -38,6 +38,23 @@
             <e-heading underline tag-name="h2" color="gray">{{ $t('c-product-detail.productDescriptionTitle') }}</e-heading>
             <div :class="b('description-text')" v-html="product.description"></div>
           </div>
+          <c-collapse-group>
+            <c-collapse v-if="product.media_attributes"
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] v-if doesn't work if its an empty array
- [ ] Use product.media_attributes.productDataSheet. I sent you an example object in slack

File: app/components/c-product-detail.vue [hunk 5]
```diff
@@ -38,6 +38,23 @@
...
+                        :class="b('documents')"
+                        :background="false"
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] Is it really false? Was this discussed with Christoph?

File: app/components/c-product-detail.vue [hunk 6]
```diff
@@ -38,6 +38,23 @@
...
+                        :title="$t('c-product-detail.productDocumentsTitle')"
+                        :active="true"
+            >
+              <c-linklist :items="product.media_attributes"/>
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] Use correct property

File: app/components/c-product-detail.vue [hunk 7]
```diff
@@ -38,6 +38,23 @@
...
+            </c-collapse>
+            <c-collapse v-if="product.product_videos"
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] Use correct property

File: app/components/c-product-detail.vue [hunk 8]
```diff
@@ -215,13 +238,33 @@
...
       color: $color-grayscale--200;
-      padding: $spacing--10 $spacing--20;
+      padding: $spacing--10 $spacing--20 $spacing--30 $spacing--20;
 
       @include media(sm) {
         padding: $spacing--10 $spacing--30 $spacing--40 $spacing--30;
       }
     }
 
+    &__documents {
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] Remove

File: app/components/c-product-detail.vue [hunk 9]
```diff
@@ -215,13 +238,33 @@
...
+      .e-heading--underline {
+        width: percentage(1);
+
+        .e-heading__inner {
+          width: percentage(1);
+        }
+      }
+    }
+
+    &__videos {
```
Comment (geroldboehler) [inline app/components/c-product-detail.vue]: - [ ] Remove

File: package.json [hunk 10]
```diff
@@ -75,6 +75,7 @@
     "axios": "~0.18.0",
     "babel-polyfill": "~6.26.0",
     "bootstrap": "~4.0.0",
+    "eslint-plugin-brackets": "~0.1.3",
```
Comment (geroldboehler) [inline package.json]: - [ ] Remove. Please be careful with packages!

File: app/components/c-linklist.vue (hunk 3 above)
Comment (adrianburtscher) [reply app/components/c-linklist.vue]: I need a css class for the list.

**Pull request closed without merging**